tableDialog_view.py

```diff
@@ -5,6 +5,8 @@
 
 
 class TableDialog(QDialog):
+    """Widget for displaying data (represented as list) in table form
```

Comment:
What table form? Maybe better to say "table forms"

Reply:
How plural noun helps? Data is represented as table, there's no other forms of representation because data is being fitted in QTableView or TableWidget.

Reply:
> How plural noun helps? Data is represented as table, there's no other forms of representation because data is being fitted in QTableView or TableWidget.

it is not clear what table form means. Therefore, if you add the plural, it will be clear that this applies to any table.
but if you do not want, do not correct)